sr_description/hand/xacro/finger/biotac/biotac.urdf.xacro

```diff
@@ -154,20 +154,20 @@ xmlns:geom="http://playerstage.sourceforge.net/gazebo/xmlschema/#geom">
         <inertia ixx="0.0" ixy="0.0" ixz="0.0" iyy="0.0" iyz="0.0"
         izz="0.0" />
       </inertial>
-      <visual>
+      <!--visual>
```

Comment:
No. Commenting out the visual did the trick. 

I made the tip follow the dummy so we get a normal chain in the srdfs:
    <group name="first_finger">
        <joint name="rh_FFJ4" />
        <joint name="rh_FFJ3" />
        <joint name="rh_FFJ2" />
        <joint name="rh_FFJ1" />
        <joint name="rh_FFtip" />
        <chain base_link="rh_palm" tip_link="rh_fftip" />
    </group>